contrib/epee/src/mlog.cpp

```diff
@@ -117,6 +124,31 @@ static const char *get_default_categories(int level)
   return categories;
 }
 
+#ifdef WIN32
```

Comment:
_WIN32 as above ?

Reply:
No need. WIN32 works after windows.h is included.